.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,62 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.223.0/containers/python-3
+{
+    "name": "Python 3",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "context": "..",
+        "args": {
+            // Update 'VARIANT' to pick a Python version: 3, 3.10, 3.9, 3.8, 3.7, 3.6
+            // Append -bullseye or -buster to pin to an OS version.
+            // Use -bullseye variants on local on arm64/Apple Silicon.
+            "VARIANT": "3.10-bullseye",
+            // Options
+            "NODE_VERSION": "lts/*"
```

Comment:
NODE_VERSION here is set to "lts/*", but [in the Dockerfile](https://github.com/jab/bidict/pull/237/files#diff-13bd9d7a30bf46656bc81f1ad5b908a627f9247be3f7d76df862b0578b534fc6R9) it's set to "none".
1. Should this be "none" here too?
2. What happens when these don't agree?
3. Can we somehow single-source this? If not, let's at least add a comment to each file like "Note: must manually keep this in sync between Dockerfile and devcontainer.json".

I see that VARIANT is also duplicated across the two files. Both of its values currently agree, but any args common to both files that should agree deserve the same treatment.

Reply:
(I speculatively pushed cc629e2 to my branch in case it helps. Should those changes be applied?)

Reply:
I believe the setting in the dockerfile is the default value, and the one in the devcontainer.json file is overriding that default. I didn't add either, this (both files) is mostly just the default python devcontainer setup you get when you add one and ask it for the python flavor.

Reply:
So no, I don't think we should take your change, this way if the templates change they'll be easier to compare